api/serverpb/server.proto

```diff
@@ -141,6 +141,16 @@ message SlackAlertingSettings {
   ];
 }
 
+// STTCheckIntervals represents intervals between STT checks.
+message STTCheckIntervals {
+  // Default check interval
```

Comment:
Comment too) 
And add dots at the end of comments, please

Reply:
Done